installer/windows/README.md

```diff
@@ -43,7 +42,34 @@ The VC++ Redistributable can be downloaded from https://www.microsoft.com/en-us/
 
 Make sure you have the 64-bit versions. Chain Core Windows does not support 32-bit. Do not actually run these installers, just provide them.
 
-### Build
+## Building the Windows installer
+
+### Using the build script
+
+Create a copy of the sample batch file `buildChainExe.bat.sample` inside the
+`installer/windows` directory. In this case, we'll call it `buildChainExe.bat`.
+
+Open `buildChainExe.bat` in your editor of choice and make the following changes:
+
+* Replace all instances of `Z:\PATH\TO\CERT` with the path to your `.pfx`
+certificate for signing binaries.
+* Replace all instances of `CERT_PASSWORD` with the password for your `.pfx` certificate.
+* Replace `VERSION` in `Chain_Core_VERSION.exe` with the `x.y.z` version of the
+Windows installer that you intend to build.
+
+From inside the `installer/windows` directory, run:
+
+```
+buildChainExe.bat
+```
+
+The resulting binary will be available at `ChainBundle/Chain_Core_[your-version].exe`.
+
+### Manual
+
+🚧 Hang on. Only go down here if you know what you're doing 🚧
```

Comment:
LOL. But also maybe make it clear that this is just the same thing as the script.